zkevm-circuits/src/evm_circuit/execution.rs

```diff
@@ -817,6 +816,19 @@ impl<F: Field> ExecutionConfig<F> {
             |mut region| {
                 let mut offset = 0;
 
+                self.advices.iter().enumerate().for_each(|(idx, &col)| {
+                    if idx < *EVM_LOOKUP_COLS {
+                        region.name_column(|| "EVM_Adv Phase3".to_string(), col)
+                    } else if idx < *EVM_LOOKUP_COLS + N_PHASE2_COLUMNS {
+                        region.name_column(|| "EVM_Adv Phase2".to_string(), col)
+                    } else {
+                        region.name_column(|| "EVM_Adv Phase1".to_string(), col)
+                    }
+                });
+                region.name_column(|| "EVM_num_rows_inv", self.num_rows_inv);
+                region.name_column(|| "EVM_rows_until_next_step", self.num_rows_until_next_step);
+                region.name_column(|| "Copy_Constr constants", self.constants);
```

Comment:
Same comment as before to avoid spaces.  These could be:
- `EVM_Adv_Phase3`
- `EVM_Adv_Phase2`
- `EVM_Adv_Phase1`
- [...]
- `Copy_Constr_constants`

Also let me do some more requests :sweat_smile: 
The `EVM_LOOKUP_COLS` are split into different sections.  Could you name them accordingly?
- `EVM_lookup_fixed_0`, `EVM_lookup_fixed_1`, ...
- `EVM_lookup_tx_0`, `EVM_lookup_tx_1`, ...
- `EVM_lookup_rw_0`, `EVM_lookup_rw_1`, ...
- and so on

After that comes the `N_PHASE2_COLUMNS`

Then `N_COPY_COLUMNS` that could be named `EVM_copy_0`, `EVM_copy_1`, ...

Then `N_BYTE_LOOKUPS` that could be named `EVM_lookup_byte_0`, `EVM_lookup_byte_1`, ...

And finally the rest are `EVM_Adv_Phase1_0`, `EVM_Adv_Phase2_1`, ...

You can see this logic here: https://github.com/privacy-scaling-explorations/zkevm-circuits/blob/fadb11b9643fd66a730db95a6561a0482991ce9e/zkevm-circuits/src/evm_circuit/util.rs#L351-L377

Reply:
Fixed in 5b537c2c.
Thanks for the suggestion!